README.md

```diff
@@ -143,6 +143,11 @@ The operator allows the administrator to deploy the [NmState State Controller](h
 This project manages host networking settings in a declarative manner. The networking state is described by a pre-defined schema. Reporting of current state and changes to it (desired state) both conform to the schema.
 Nmstate is aimed to satisfy enterprise needs to manage host networking through a northbound declarative API and multi provider support on the southbound. NetworkManager acts as the main (and currently the only) provider supported.
 
+It communicate with a NetworkManager instance running on the node using D-Bus. Make sure that NetworkManager is installed and running on each node.
+
+yum install NetworkManager
```

Comment:
Can you please add this as a code section?

Reply:
Done thanks!